snowpack/src/commands/install.ts

```diff
@@ -92,7 +92,7 @@ function resolveWebDependency(dep: string): DependencyLoc {
     const isJSFile = ['.js', '.mjs', '.cjs'].includes(path.extname(dep));
     return {
       type: isJSFile ? 'JS' : 'ASSET',
-      loc: require.resolve(dep, {paths: [cwd]}),
+      loc: fs.realpathSync.native(require.resolve(dep, {paths: [cwd]})),
```

Comment:
If we decide to go ahead with this change, I'd suggest to add a comment above this line, referencing this pull request. I'm sure folks reading the code will wonder about what this is and why we use it, the PR has a great explanation

Reply:
Done.